pkg/controller/command/verifiable/command_test.go

```diff
@@ -1237,6 +1237,7 @@ func TestGeneratePresentationHelperFunctions(t *testing.T) {
 		credList := make([]json.RawMessage, 0)
 
 		req := &PresentationRequest{
+			ProofOptions:          &ProofOptions{SignatureType: Ed25519Signature2018},
```

Comment:
@rolsonquadras @sudeshrshetty @llorllale fyi

Reply:
It is ok. I have changed the order of conditions (due not nestif).